Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "verkle_tree_img"
+version = "0.1.0"
+edition = "2021"
+repository = "https://github.com/blockscout/blockscout-verkle-tree"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4"
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
+env_logger = "0.9"
+log = "0.4"
+config = "0.13"
+serde = { version = "1", features = ["derive"] }
+clap = { version = "3", features = ["derive"] }
```

Comment:
let's sort them )  I know that they are not sorted in `blockscout-rs`, but I will fix it soon